packages/jest-snapshot/src/InlineSnapshots.ts

```diff
@@ -49,10 +49,15 @@ export function saveInlineSnapshots(
   snapshots: Array<InlineSnapshot>,
   prettierPath: Config.Path,
 ): void {
-  const prettier = prettierPath
-    ? // @ts-expect-error requireOutside Babel transform
-      (requireOutside(prettierPath) as Prettier)
-    : undefined;
+  let prettier;
+  if (prettierPath) {
+    try {
+      // @ts-expect-error requireOutside Babel transform
+      prettier = requireOutside(prettierPath) as Prettier;
+    } catch (_) {
```

Comment:
```suggestion
    } catch {
```

Reply:
Haha, TIL you can use no arguments. Didn't realize you just have to leave off the parentheses. thx.